motionplan/tpSpaceRRT.go

```diff
@@ -177,18 +165,29 @@ func (mp *tpSpaceRRTMotionPlanner) plan(ctx context.Context,
 	seedPos := spatialmath.NewZeroPose()
 
 	startNode := &basicNode{q: make([]referenceframe.Input, len(mp.frame.DoF())), cost: 0, pose: seedPos, corner: false}
-	goalNode := &basicNode{q: make([]referenceframe.Input, len(mp.frame.DoF())), cost: 0, pose: goal, corner: false}
+	maps := &rrtMaps{startMap: map[node]node{startNode: nil}}
```

Comment:
This is necessary since we are no longer ever using unidirectional solving for position only mode